src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyen_checkout/checkoutConfiguration.js

```diff
@@ -267,9 +267,15 @@ function getGiftCardConfig() {
 
 function handleOnChange(state) {
   let { type } = state.data.paymentMethod;
+  const multipleTxVariantsMethods = constants.MULTIPLE_TX_VARIANT_METHODS;
   if (store.selectedMethod === 'googlepay' && type === 'paywithgoogle') {
     type = 'googlepay';
   }
+
+  if (multipleTxVariantsMethods.includes(store.selectedMethod)) {
```

Comment:
Can't we just use the `store.selectedMethod` as key for the `store.componentsObj` for all payment methods? This way we don't have to maintain an array with "multipleTxVariantsMethods" and it will be compatible out of the box for future components with multiple tx variants.